### ISSUE TITLE
`appearance` is not implemented

### ISSUE THREAD

**Issue by out-of-phaze** [OpenDreamProject/OpenDream#1090]
SSoverlays on Nebula (and probably every other codebase) doesn't work because `appearance` is not implemented, meaning things like appearance copies don't work.
Implementing some features of `appearance` (which is apparently a `/image`?) would go a long way towards improving the renderer. The most important things I can think of are:
- [x] Appearance reading (`var/image/appearance = atom.appearance`)
- [x] Appearance copy (`atom.appearance = appearance`) (#1542)
- [x] Implicit appearance copy (`atom.appearance = atom`) (#1542)
- [ ] Appearance `overlays` variable access

**Comment by out-of-phaze** [comment]
The fourth part is still an issue.
```
[ERRO] world.log: Exception occurred: Invalid appearance var "overlays"
=DM StackTrace=
   /datum/controller/subsystem/zcopy/fixup_appearance_planes code/controllers/subsystems/zcopy.dm:629
   /datum/controller/subsystem/zcopy/tick_mimic code/controllers/subsystems/zcopy.dm:471
   /datum/controller/subsystem/zcopy/fire code/controllers/subsystems/zcopy.dm:224
   /datum/controller/subsystem/zcopy/Initialize code/controllers/subsystems/zcopy.dm:169
   /datum/controller/subsystem/DoInitialize code/controllers/subsystem.dm:174
   /datum/controller/master/Initialize code/controllers/master.dm:183

=C# StackTrace=
System.Exception: Invalid appearance var "overlays"
   at OpenDreamRuntime.Procs.DMProcState.ThrowInvalidAppearanceVar(String field) in c:\Users\User\phaze\OpenDream\OpenDreamRuntime\Procs\DMProc.cs:line 890
   at OpenDreamRuntime.Procs.DMProcState.DereferenceField(DreamValue owner, String field) in c:\Users\User\phaze\OpenDream\OpenDreamRuntime\Procs\DMProc.cs:line 872
   at OpenDreamRuntime.Procs.DMOpcodeHandlers.DereferenceField(DMProcState state) in c:\Users\User\phaze\OpenDream\OpenDreamRuntime\Procs\DMOpcodeHandlers.cs:line 1557
   at OpenDreamRuntime.Procs.DMProcState.Resume() in c:\Users\User\phaze\OpenDream\OpenDreamRuntime\Procs\DMProc.cs:line 415
   at OpenDreamRuntime.DreamThread.Resume() in c:\Users\User\phaze\OpenDream\OpenDreamRuntime\DreamThread.cs:line 223
```
`if (appearance:overlays:len)` is runtiming.

**Comment by amylizzle** [comment]
#2104 close this?

**Comment by out-of-phaze** [comment]
Fixed by #2104.
